tests/system/providers/google/README.md

```diff
@@ -0,0 +1,99 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+-->
+
+# Google provider system tests
+
+## Tests structure
+
+All Google-related system tests are located inside this subdirectory of system tests which is
+`tests/system/providers/google/`. They are grouped in directories by the related service name, e.g. all BigQuery
+tests are stored inside `tests/system/providers/google/bigquery/` directory. In each directory you will find test files
+as self-contained DAGs (one DAG per file). Each test may require some additional resources which should be placed in
+`resources` directory found on the same level as tests. Each test file should start with prefix `example_*`. If there
+is anything more needed for the test to be executed, it should be documented in the docstrings.
+
+Example files structure:
+
+```
+tests/system/providers/google
+├── bigquery
+│   ├── resources
+│   │   ├── example_bigquery_query.sql
+│   │   └── us-states.csv
+│   ├── example_bigquery_queries.py
+│   ├── example_bigquery_operations.py
+.   .
+│   └── example_bigquery_*.py
+├── dataflow
+├── gcs
+.
+└── *
+```
+
+## Initial configuration
+
+Each test requires some environment variables. Check how to set them up on your operating system, but on UNIX-based
+OSes this should work:
+
+```commandline
+export NAME_OF_ENV_VAR=value
+```
+
+To confirm that it is set up correctly, run `echo $NAME_OF_ENV_VAR` which will display its value.
+
+### Required environment variables
+
+- `SYSTEM_TESTS_GCP_PROJECT` - GCP project name that will be used to run system tests (this can be checked on the UI
+  dashboard of the GCP or by running `gcloud config list`).
+
+- `SYSTEM_TESTS_ENV_ID` - environment ID that is unique across between different executions of system tests (if they
```

Comment:
Nit:

"`unique across between ...`"

Use either "across" or "between", having both makes the sentence a bit odd.